### DISCUSSION TITLE
PLEASE TELL ME YOUR FAVORITE ARTIST!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by Massan0917** [NAIST-SE/AttractingContributors#831] [2023 Attraction Contributions]
please tell me your favorite artist!!!!!!!!!
https://github.com/Massan0917/AttractingContributors/
https://github.com/Massan0917/AttractingContributors/
https://github.com/Massan0917/AttractingContributors/

**Comment by KKKKKXY** [comment]
I added my favorite artist into your list.(githubid: KKKKKXY)
